### PR TITLE
WebNN support check in home page

### PR DESCRIPTION
**Pull request by ibelem**
Submite this PR based on the user experience feedback.

- Putting in checks directly in the WebNN Developer Preview home page to make sure that the developer has enabled the webnn flag
- If user happens to use non-Edge browser, user could go to Installation Guides for non-Edge browser
- Show following infomation above the links in page footer: 
webnn_status

@fdwr PTAL

### CODE REVIEW

**Comment by ibelem** [comment]
@fdwr Please also help to check if this PR is acceptable for you. :)

File: assets/css/style.css [hunk 1]
```diff
@@ -416,6 +416,45 @@ footer ul li a:hover {
...
+
+#status.red #support, #status.red #installation {
+  background-color: var(--pink);
+  color: white;
+}
+
+#status.red>a {
+  background-color: var(--pink);
+  color: white;
+  padding: 2px 6px 1px 6px;
```
Comment (fdwr) [inline assets/css/style.css:441]: Would it be more accessible to use em and rem than px?
(I won't block on this - can be done in a separate CR if prudent)

File: assets/js/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { webNnStatus } from './common_utils.js'
+
+async function webNNChecker() {
+  const webnnStatus = await webNnStatus();
```
Comment (fdwr) [inline assets/js/index.js:4]: (nit)
`webnnStatus`
`webNnStatus`

**Review by fdwr — APPROVED** [review]
👍 Good idea.

File: assets/css/style.css (hunk 1 above)
Comment (ibelem) [reply assets/css/style.css:441]: Thanks for pointing this out, use em/rem for the whole css file need more efforts.

File: assets/js/index.js (hunk 2 above)
Comment (ibelem) [reply assets/js/index.js]: Thanks for the feedback, fixed in new commit https://github.com/microsoft/webnn-developer-preview/pull/43/commits/9636b15aae9e0e02096b2c3e78f8bcbdc8ffb1f2 , PTAL

**Review by fdwr — APPROVED** [review]
👍

**Pull request merged**